src/cluster.cpp

```diff
@@ -47,29 +47,29 @@ namespace
 {
 
 std::unique_ptr<IStreamReader>
-getClusterReader(const Reader& zimReader, offset_t offset, Compression* comp, bool* extended)
+getClusterReader(const Reader& zimReader, offset_t offset, Cluster::Compression* comp, bool* extended)
 {
   uint8_t clusterInfo = zimReader.read(offset);
   // Very old zim files used 0 as a "default" compression, which means no compression.
   uint8_t compInfo = clusterInfo & 0x0F;
   if(compInfo == 0) {
-    *comp = Compression::None;
+    *comp = Cluster::Compression::None;
   } else if (compInfo == 2 /* Zip compression */) {
     throw std::runtime_error("zlib not enabled in this library");
   } else if (compInfo == 3 /* Bzip2 compression */) {
     throw std::runtime_error("bzip2 not enabled in this library");
```

Comment:
We should use `Cluster::Compression` enum here.

Reply:
Fixed